### ISSUE TITLE
Typos in english

### ISSUE THREAD

**Issue by simonporte** [persepolisdm/persepolis#429]
I started french translation on transifex, and I noticed some typos.
As I don't have much clue where to find them in source (except via file search) I will post them here.
If there is a better place to do so, or an easy way to correct this by myself, please tell me.
I'll use transifex IDs to identify the strings.

- [x] 8 - "Ok" should be full caps like in the other occurrences of the word
- [ ] 91 - Persepolis is trying to connect!<missing_space>be patient!
- [x] 93, 214 - "Try agian!" Obvious typo
- [x] 27, 204, 225 - KB/S should be kB/S, k is a SI prefix, unlike Ki for Kibi which has a initial capital letter. In fact, it is kB/s with a lowercase s, see next point.
- [x] 27, 28, 204, 205, 225, 226 - <unit_prefix>B/S should be <unit_prefix>B/s, the "second" symbol is SI notation, and lowercase.
- [x] 115 - " is already existed! - It does not seem to be a correct sentence, use exists, or existed, but not is existed.
- [x] `10
11
12
15
16
17
20
21
23
29
40
41
48
49
73
78
79
80
81
216
217
218
219
220
221
236
238
240
242
244
245
252
253
257
259
260
261
262
263
264
265
275
292
301
303
304
305
308
309
311
313` - I'm not native speaker but I think in english there is no space before colon. Problem present on a lot of strings.
- [x] 51, 89, 98, 236, 324 - Missing capital letter
- [x] 103, 106, 111, 114 - "first", Missing capital letter

**Comment by hayyan71** [comment]
@mdmk Thanks.

The devs are not native English speaker, too! So, you can see such typing mistakes in many phrases in Persepolis codes.
Translator's cannot edit source strings directly. if you see such typing mistakes, please report them in Github page.

I'll fix some of the problems you've mentioned in the next update.

Regards

**Comment by hayyan71** [comment]
@mdmk

**Comment by hayyan71** [comment]
@mdmk 
89, 98, 103,106, 111, 114: I'll add some comments in Transifex for these lines later. they need a bit more descriptions.

**Comment by hayyan71** [comment]
added new pull request #431

**Comment by simonporte** [comment]
@hayyan71 
You're welcome, thank you for doing the modifications in sources.

**Comment by hayyan71** [comment]
@mdmk 

updated source strings. you may check transifex.com now.

**Comment by simonporte** [comment]
- [ ] 1. It seems there is still an occurrence of "Persepolis is trying to connect!<missing_space>be patient!" 
- [x] 2. and two of "Try agian!", 
- [ ] 3. and "close" doesn't have a capital letter.

**Comment by simonporte** [comment]
New suggestions more or less sorted by order of relevance:
I don't guarantee the correctness of Transifex IDs (I noted some a few days ago)
- [x] 176 - Remove selected downloads form list - should be from not form
- [x] 223 - Restart Persepolis Please!Some changes take effect after restarting persepolis (missing capital letter for Persepolis)
- [x] 151 - Download video or audio from youtube and ... (missing capital letter for Youtube)
- [x] 75 - Reconnecting aria2...   (missing capital letter for Aria2)
- [x] 238 - Change aria2 default path (again missing capital letter for Aria2)
- [x] 58 - Copy  selected to clipboard (two spaces between Copy and selected)
- [x] time out / timeout inconsistency the two are used
- [x] 66 - Please wait ... (there should be no space before ...)
- [ ] 17, 19, 20, 292, 294, 295 - UserName should not have a N capital letter
- [ ] 15, 19, 21, 290, 294, 296 - PassWord should not have a W capital letter
- [x] 252 - ToolBar should not have a B capital letter
- [x] Show download complete dialog,when download has finished. (At least there should be a space after comma, but I think there should be no comma at all ? It's the same sentence in one block)
- [ ] 214 - Download information (information should be plural isn't it ?)

**Comment by hayyan71** [comment]
`17, 19, 20, 292, 294, 295 - UserName should not have a N capital letter`
 
`15, 19, 21, 290, 294, 296 - PassWord should not have a W capital letter`

I'm not sure about these lines. I have no idea why some developers write N in "UserName" and W in "PassWord" in capital letters. (this is not the first time I see this.)

**Comment by hayyan71** [comment]
new pull request: #439

**Comment by simonporte** [comment]
Pretty fast, thank you for editing all these!

You think there may be a reason other than better readability?
It is not common to write words this way.
Any way to ask the other devs for explanation on the matter?
At least it is not important, it is just style choice/problem, not a typo or incorrect sentence.

**Comment by simonporte** [comment]
You did a great job @hayyan71 👍 
I checked Transifex again and completed the new and modified strings.
And I ticked what was done upper on the thread.
I've seen two more things:
103 - Persepolis is trying to connect! *b*e patient! (missing capital letter on "be")
Sorry to make you edit this string twice I did not see it before.
156 - Download video or audio from Youtube, Vimeo, etc...
I'm sorry, I'm very picky on this, but there should be only etc. (with a dot) or ..., not both, it is redundant.
Feel free to ignore this I it bothers you.

Old ones you did not correct:
97 - It seems there is still an occurrence of "Persepolis is trying to connect!<missing_space>be patient!"
60 - and "close" doesn't have a capital letter.
221 - Download information (information should be plural isn't it ?)
